src/rules/function-url-no-scheme-relative/README.md

```diff
@@ -0,0 +1,43 @@
+# function-url-no-scheme-relative
+
+Disallow scheme relative urls.
```

Comment:
How about adding this as the first time below the example?:

"A [scheme-relative url](https://url.spec.whatwg.org/#syntax-url-scheme-relative) is a url that begins with `//` followed by a host."